tests/qos/test_tunnel_qos_remap.py

```diff
@@ -114,7 +122,14 @@ def test_encap_dscp_rewrite(ptfhost, upper_tor_host, lower_tor_host,
         # Send original packet
         testutils.send(ptfadapter, src_port, pkt)
         # Verify encaped packet
-        testutils.verify_packet_any_port(ptfadapter, expected_pkt, dst_ports)
+        try:
+            testutils.verify_packet_any_port(ptfadapter, expected_pkt, dst_ports)
+            print("Verified DSCP combination {}".format(str(dscp_combination)))
+        except AssertionError:
+            print("Failed to verify packet on DSCP combination {}".format(str(dscp_combination)))
```

Comment:
Please use `logger.info` instead of `print` so that we can capture the output in test log.